pkg/engine/params.go

```diff
@@ -78,6 +78,9 @@ func getParameters(cs *api.ContainerService, generatorCode string, aksEngineVers
 	}
 	if properties.HostedMasterProfile != nil {
 		addValue(parametersMap, "masterSubnet", properties.HostedMasterProfile.Subnet)
+		if len(properties.AgentPoolProfiles) > 0 && len(properties.AgentPoolProfiles[0].VnetCidrs) > 0 {
```

Comment:
Is it safe to hardcode the values from the first agent pool profile?
Does AKS only create one agent pool at a time (and will this continue to be the case)?

If it is safe can we add a comment here?

Reply:
@jackfrancis or @mboersma for another set of eyes. 
I'm not familiar with how aks-engine is used in AKS.

Reply:
AKS only supports to use one vnet. When a user creates a new k8s cluster, a custom vnet can be specified and the cidr of the agent pool will be set. If the custom vnet is not specified, DefaultVNETCIDR will be set. A cluster must have at least one agent pool so anyway the VnetCidrs of the default agent pool (index: 0) should be set.